config/install/core.entity_form_display.node.starships.default.yml

```diff
@@ -0,0 +1,139 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.starships.field_cargo_c
+    - field.field.node.starships.field_consum
+    - field.field.node.starships.field_cost_in_credits
+    - field.field.node.starships.field_crew
+    - field.field.node.starships.field_films
+    - field.field.node.starships.field_hyperdrive_rating
+    - field.field.node.starships.field_length
+    - field.field.node.starships.field_manufacturer
+    - field.field.node.starships.field_max_atmospheric_speed
+    - field.field.node.starships.field_model
+    - field.field.node.starships.field_passen
+    - field.field.node.starships.field_pilots
+    - field.field.node.starships.field_starship_c
+    - node.type.starships
+id: node.starships.default
+targetEntityType: node
+bundle: starships
+mode: default
+content:
```

Comment:
The Starships content type is missing the URL field.

Reply:
Wow. It sure is. Or should I say was. It is fixed now.